ProGet/installation/installation-guide/traditional-installer.md

```diff
@@ -1,108 +1,175 @@
-
-![](/resources/documentation/proget-installation/progetwebserver.png){.screenshot}
-
-If IIS is installed on the server, you will have the option of installing ProGet in IIS. This will create an application pool and website in the local IIS configuration.
-
-## 11. User Account {#user-account data-title="11. User Account"}
-
-By default, the installer will use the Network Service account to run the ProGet Service and Web Application. We Recommend sticking with this, and changing the account later if you need to.
-
-![](/resources/documentation/proget-installation/useraccount.png){.screenshot}
-
-## 12. Summary {#summary data-title="12. Summary"}
-
-Make sure everything here looks OK, and then click Install.
-
-![](/resources/documentation/proget-installation/summary.png){.screenshot}
-
-If any of the items indicate an error, you can click on it to go back to that page and fix it.
-
-## 13. Patiently Wait {#patiently-wait data-title="Patiently Wait"}
-
-If you chose to have SQL Server Express installed, then the actual installation process will take a while, as the installer must download the SQL installer and then wait for it to complete. On some servers, we've seen this take as long as fifteen minutes.
-
-![](/resources/documentation/proget-installation/progetclickinstall.png){.screenshot}
-
-Clicking cancel (especially during the SQL Express installation) is not advised, as you may have to manually remove some bits of SQL Server using Add/Remove programs. Once everything's done, you'll be ready to start using ProGet.
+*We are planning to stop shipping legacy installer in 2021*
+
+For the most convenient way to install ProGet is to use the [InedoHub](https://my.inedo.com/downloads). It is primarily used to install, update, and otherwise service applications in the Inedo Suit. You can learn more about it on how the InedoHub can help you save time by reading our [guide](https://docs.inedo.com/docs/desktophub/overview).
+
+Please also note that if you have the InedoHub installed you will NOT be able to use the traditional install method without first completely removing the InedoHub and any products installed using it.
+
+If you require a legacy installer for any version of ProGet, you can on the [ProGet All Version](https://my.inedo.com/proget/versions) page, and clicking on the “Traditional Installer” for the version you want.
+
+Once you have downloaded and opened the installer, you’ll be asked to review and accept out [license agreement](https://inedo.com/proget/license-agreement) before you can use ProGet
+
+Complete the rest of the installation by selecting each setting appropriate for your system. We have updated our [detailed installation guide](https://docs.inedo.com/docs/proget/installation/installation-guide) to reflect the new streamlined layout of the InedoHub, but the options are still the same.
```

Comment:
Although the [ProGet installation guide](/docs/proget/installation/installation-guide) uses the Inedo Hub, the settings are mostly the same.